lighthouse-core/config/config-plugin.js

```diff
@@ -156,6 +204,7 @@ class ConfigPlugin {
     }
 
     const {
+      groups: pluginGroupsJson,
```

Comment:
total nit, but put `groups` after `category` (and `categories` in the `return` below) to match the order of the type declarations